com.unity.renderstreaming/Tests/Runtime/SignalingHandlerTest.cs

```diff
@@ -0,0 +1,307 @@
+
+    class InputChannelTest : DataChannelBase
+    {
+        public void SetLocal(bool isLocal)
+        {
+            Type myClass = typeof(InputChannelTest);
+            FieldInfo fieldLocal = myClass.GetField("local",
+                BindingFlags.Public | BindingFlags.Instance | BindingFlags.NonPublic);
+            fieldLocal.SetValue(this, true);
+        }
+
+
+        public void SetLabel(string label)
+        {
+            Type myClass = typeof(InputChannelTest);
+            FieldInfo fieldLabel = myClass.GetField("label",
+                BindingFlags.Public | BindingFlags.Instance | BindingFlags.NonPublic);
+            fieldLabel.SetValue(this, label);
+        }
+    }
+
+    class StreamSourceTest : StreamSourceBase
+    {
+        private Camera m_camera;
+
+        protected override MediaStreamTrack CreateTrack()
+        {
+            m_camera = gameObject.AddComponent<Camera>();
+            return m_camera.CaptureStreamTrack(256, 256, 0);
+        }
+    }
+
+    class StreamReceiverTest : StreamReceiverBase
+    {
+    }
+
+    class TestContainer<T> : IDisposable where T : SignalingHandlerBase, IMonoBehaviourTest
+    {
+        public MonoBehaviourTest<T> test;
+        public RenderStreamingInternal instance;
+        public SignalingEventProvider provider;
+
+        private static RenderStreamingDependencies CreateDependencies(MonoBehaviour behaviour)
+        {
+            return new RenderStreamingDependencies
+            {
+                signaling = new MockSignaling(),
+                config = new RTCConfiguration
+                {
+                    iceServers = new[] { new RTCIceServer { urls = new[] { "stun:stun.l.google.com:19302" } } },
+                },
+                encoderType = EncoderType.Software,
+                startCoroutine = behaviour.StartCoroutine
+            };
+        }
+
+        public static TestContainer<T> Create(string name)
+        {
+            var test = new MonoBehaviourTest<T>();
+            var dependencies = CreateDependencies(test.component);
+            var instance = new RenderStreamingInternal(ref dependencies);
+            var provider = new SignalingEventProvider(instance);
+            var container = new TestContainer<T> { test = test, instance = instance, provider = provider };
+            test.component.SetHandler(instance);
+            test.gameObject.name = name;
+            provider.Subscribe(test.component);
+            return container;
+        }
+
+        public void Dispose()
+        {
+            instance.Dispose();
+            test.component.StopAllCoroutines();
+            UnityEngine.Object.Destroy(test.gameObject);
+        }
+    }
+
+    [UnityPlatform(exclude = new[] {
+        RuntimePlatform.OSXEditor,
+        RuntimePlatform.OSXPlayer,
+        RuntimePlatform.LinuxEditor,
+        RuntimePlatform.LinuxPlayer
+    })]
+    [ConditionalIgnore(ConditionalIgnore.IL2CPP, "Process.Start does not implement in IL2CPP.")]
```

Comment:
if use `MockSignaling`, I think no need ignore test. 
`MockSignaling` not use `Process` class.

Reply:
OK, I will fix it in the next PR.